libraries/radwordsprocessing/editing/find-and-replace.md

```diff
@@ -1,44 +1,67 @@
 
+* **FindAll(Regex regex):** Finds all matches of the passed **Regex**.
 
-Finding all instances of a string and replacing it with another can be achieved with the __ReplaceText()__ method of __RadFlowDocumentEditor__. The editor exposes three methods accepting different parameters.
+Both methods return a collection of **FindResult** instances, which in turn expose the following properties:
 
+* **Runs**: Gets a collection of Runs that contains the searched text.
 
-* __void ReplaceText(string oldText, string newText):__ Replaces all occurrences of the *oldText* string with the *newText* string. Matches the casing of the old text and replaces occurrences within a word as well.
+* **RelativeStartIndex**: Gets the index of the first character in the searched text inside the first Run.
 
-* __void ReplaceText(string oldText, string newText, bool matchCase, bool matchWholeWord):__ Replaces all occurrences of a string with another string. Accepts Boolean parameters which specify whether the casing should be matched and only whole words should be replaced.
+* **RelativeEndIndex**: Gets the index of the last character in the searched text inside the last Run.
 
-* __void ReplaceText(Regex regex, string newText):__ Replaces all matches of the passed __Regex__ with the new text.
+* **FullMatchText**: Gets the matched text.
 
+**Example 1** shows how to create a **RadFlowDocumentEditor** instance and use it to find all matches of the word "code".
 
-__Example 1__ shows how to create a __RadFlowDocumentEditor__ instance and use it to replace all matches of the word "code" with the phrase "source code".
+#### **[C#] Example 1: Find text**
 
+{{region cs-radwordsprocessing-editing-find-and-replace_0}}
 
-#### __[C#] Example 1: Replace text__
+	RadFlowDocumentEditor editor = new RadFlowDocumentEditor(document); // document is an instance of the RadFlowDocument class
+	ReadOnlyCollection<FindResult> findResults = editor.FindAll("code", matchCase: true, matchWholeWord: true);
+{{endregion}}
 
-{{region cs-radwordsprocessing-editing-find-and-replace_0}}
+## Replace Text
+
+Finding all instances of a string and replacing it with another can be achieved with the **ReplaceText()** method of **RadFlowDocumentEditor**. The editor exposes two methods accepting different parameters.
+
+
+* **void ReplaceText(string oldText, string newText, bool matchCase, bool matchWholeWord):** Replaces all occurrences of a string with another string. Accepts two optional Boolean parameters which specify whether the casing should be matched and only whole words should be replaced. If not set the default values are respectively _true_ for matchCase and _false_ for matchWholeWord.
+
+* **void ReplaceText(Regex regex, string newText):** Replaces all matches of the passed **Regex** with the new text.
```

Comment:
* **void ReplaceText(Regex regex, string newText):** Replaces all matches of the specified **Regex** with the new text.